content/docs/command-reference/import-db.md

```diff
@@ -0,0 +1,153 @@
+<admon type="warn" title="Security Alert">
+Configure password in a local config file, separately from the connection string.
+</admon>
+
+You can pass `--conn <name>` in addition with `--sql <query>` to use that
+database connection.
+
+```dvc
+dvc import-db --sql 'select * from table' --conn pgsql
+```
+
+In addition to a connection string, DVC needs a driver to connect to the
+database. Check [Installing database drivers](#installing-database-drivers) for
+connection string format and necessary driver for your specific database.
+
+[`db`]: /doc/user-guide/project-structure/configuration#db
+
+## Installing database drivers
+
+DVC does not come preinstalled with all the drivers for databases, you’ll need
+to install the required packages for the database you want to use.
+
+Some of the recommended packages are shown below, with their expected connection
+strings:
+
+| **Database**        | **PyPI package**                  | **Connection String**                                                                               |
+| ------------------- | --------------------------------- | --------------------------------------------------------------------------------------------------- |
+| **Amazon Redshift** | `sqlalchemy-redshift`             | `redshift+psycopg2://{username}:{password}@{aws_endpoint}:5439/{database_name}`                     |
+| **Big Query**       | `pip install sqlalchemy-bigquery` | `bigquery://{project_id}`                                                                           |
+| **Databricks**      | `databricks-sql-connector`        | `databricks://token:{token}@{hostname}:{port}/{database}?http_path={http_path}`                     |
+| **MySQL**           | `mysqlclient`                     | `mysql://{username}:{password}@{hostname}/{database_name}`                                          |
+| **Oracle**          | `cx_Oracle`                       | `oracle://{username}:{password}@{hostname}/{database_name}`                                         |
+| **PostgreSQL**      | `psycopg2`                        | `postgresql://{username}:{password}@{hostname}/{database_name}`                                     |
+| **Snowflake**       | `snowflake-sqlalchemy`            | `snowflake://{user}:{password}@{account}.{region}/{database}?role={role}&amp;warehouse={warehouse}` |
+| **SQLite**          | -                                 | `sqlite://path/to/file.db`                                                                          |
+| **SQL Server**      | `pyodbc`                          | `mssql+pyodbc://{username}:{password}@{hostname}:{port}/{database_name}`                            |
+| **Trino**           | `trino`                           | `trino://{username}:{password}@{hostname}:{port}/{catalog}`                                         |
+
+DVC uses `sqlalchemy` internally. So DVC should support any SQL databases that
+provide dialects for SQLAlchemy. Refer to their documentation for more details.
```

Comment:
```suggestion
provide dialects for SQLAlchemy. Refer to their [documentation](https://docs.sqlalchemy.org) for more details.
```

Reply:
With "their", I mean the particular plugins for the database.